app/Services/PostService.php

```diff
@@ -65,7 +65,7 @@ public function update($signup, $data, $transactionId)
         // @TODO: This will is only temporary and will be removed!
         // If this is a signup update, get the most recent post.
         // If there is a quantity_pending, this is a signup.
-        if ($post->quantity_pending) {
+        if ($post->quantity || $post->quantity_pending) {
```

Comment:
hmm this seems like legacy stuff that can be removed, as the @todo mentions but I wasn't sure either just wanted to make sure that `quantity` and `quantity_pending` was accounted for. I guess we can remove it now.